Check more IPA-related services

fcami opened the pull request:
Check the status of ipa-custodia, ipa-dnskeysyncd,
ipa-otpd, kadmin and sssd.

This depends on these services being known to ipaplatform: https://github.com/freeipa/freeipa/pull/3075

rcritten commented:
LGTM. Let's let the patch land upstream. May need to work out how we check for these or go ahead and create a 4-7 branch now to line up with what is in current releases freeIPA.

fcami commented:
Added exception handling for when services are not yet known to ipaplatform.
Added logging.
Output looks fine against freeipa-server-4.7.2-1.1.fc29.x86_64 which is missing (obviously) the changes from https://github.com/freeipa/freeipa/pull/3075

fcami commented:
https://github.com/freeipa/freeipa/pull/3075 hit master
